ddtrace/ext/ci_visibility/api.py

```diff
@@ -239,7 +239,7 @@ def finish(
     @_catch_and_log_exceptions
     def get_settings(item_id: Optional[CISessionId] = None):
         log.debug("Getting settings for session %s", item_id)
-        core.dispatch_with_results("ci_visibility.session.get_settings", (item_id,))
+        core.dispatch("ci_visibility.session.get_settings", (item_id,))
```

Comment:
The actual fix here is to actually return the settings.

Reply:
Actually, since I don't have a use case for this at the moment, I'm just going to delete it.